Tests/SMP Tests.swift

```diff
@@ -168,7 +167,24 @@ public class SMP_Tests
 		{
 			precondition(n.description == BInt(n).description)
 		}
-
+        
+        //
+        ////
+        //////
+        //"MARK": - Hexadecimal
+        //////
+        ////
+        //
+        
+        XCTAssert(BInt("ffff",16) == 65535)
+        XCTAssert(BInt("ff",16) == 255.0)
+        XCTAssert(BInt("ff",16) != 100.0)
+        XCTAssert(BInt("ffff",16)! > 255.0)
+        XCTAssert(BInt("f",16)! < 255.0)
+        XCTAssert(BInt("0",16)! <= 1.0)
+        XCTAssert(BInt("f")! >= 1.0)
+        
```

Comment:
what happens when radix isn't 16? what happens if the string isn't hex?

Reply:
Got it. Will add cases gradually, I can only put only so much hours on this =)